packages/opentelemetry-cloud-trace-exporter/test/exporter.test.ts

```diff
@@ -75,21 +100,58 @@ describe('Google Cloud Trace Exporter', () => {
         }
       );
 
-      sinon.replace(
-        TraceExporter['_cloudTrace'].projects.traces,
-        'batchWrite',
-        /* tslint:disable-next-line:no-any */
-        batchWrite as any
-      );
-
       sinon.replace(exporter['_auth'], 'getClient', () => {
         if (getClientShouldFail) {
           throw new Error('fail');
         }
         /* tslint:disable-next-line:no-any */
-        return {} as any;
+        return mockClient as any;
       });
 
+      sinon.stub(protoloader, 'loadSync');
+
+      createSsl = sinon
+        .stub(grpc.credentials, 'createSsl')
+        .returns(mockChannelCreds);
+
+      createFromGoogleCreds = sinon
+        .stub(grpc.credentials, 'createFromGoogleCredential')
+        .returns(mockCallCreds);
+
+      combineChannelCreds = sinon
+        .stub(grpc.credentials, 'combineChannelCredentials')
+        .returns(mockCombinedCreds);
+
+      sinon.replace(
+        grpc,
+        'loadPackageDefinition',
+        (): grpc.GrpcObject => {
+          traceServiceConstructor = sinon.spy(
+            (host: string, creds: grpc.ChannelCredentials) => {}
+          );
+          /* tslint:disable-next-line:no-any */
+          const def: any = {
+            google: {
+              devtools: {
+                cloudtrace: {
+                  v2: {},
+                },
+              },
+            },
+          };
+          // Replace the TraceService with a mock TraceService
+          def.google.devtools.cloudtrace.v2.TraceService = class MockTraceService
+            implements TraceService {
+            /* tslint:disable-next-line:variable-name */
+            BatchWriteSpans = batchWrite;
```

Comment:
is this why you need that `bind()` or is it needed for the actual call?

Reply:
Its needed for the actual call. Without it there is an undefined error within the grpc module